hips/hip-0012.md

```diff
@@ -1,38 +1,225 @@
+So this proposal attempts to spell out a formal process for a sucessful Helm 4 release.
+
 Further criticism was that it was unclear who was "running the show" and how they were making decisions.
 Finally, we were criticized for not explaining the criteria we were using for determining when Helm 3 was complete.
-This document is intended to address these criticisms.
-It describes the process in detail, providing both a method for us to follow and a point of reference for community members to read.
+
+This document is intended to address these criticisms and present a reasonable plan for timely delivery of a valuable Helm 4 release to the community.
+It describes the process in detail, providing both a method for us to follow and a point of reference for community members to follow for contributing changes.
+
+The process based on the following ideas:
+
+- Requiring changes to be documented (either Helm 4 proposals, or standard HIPs).
+Both allows Helm communitity members to review changes, and allow Helm maintainers to provide oversight on the functional and architectual implementation of the given change. Ensuring changes are in good standing with respect to this HIP.
+Finally, documented changes should allow contributors to independently execute changes to the agreed proposal in coordination with the release engineer.
+
+- Requiring a timeboxed development effort intends to both promote focus on the most valuable changes that the community/maintainers can deliver within a reasonable timeframe.
+And avoid an "open ended" development cycle that might drag on and/or fizzle out.
+Ideally the majority of changes will be proposed and agree upon early in the development cycle (the planning phase).
+But changes to this initial scope can be ammended during the course of the development lifecycle.
+So long as they can fit within the agreed upon timeframe.
+Making amending the scope an exlicit requirement seeks to ensure that any changes are well thought out.
+For further changes that don't make it into Helm 4, there will be a Helm 5 of course!
+
 
 ## Specification
 
+### Prerequisites
+Before Helm can start the development of Helm v4 there are some things the Helm organization needs to have in order.
+These include:
+
+- Active maintainers who will take the time to review changes.
+Helm has been in an unoffical "maintenance mode" for some time where changes have slowed down and the project needs the people capable of driving a Helm 4.
+- Direction on what is in and out of scope for Helm 4.
+It’s important to be able to know what to say “yes” to and what to say “no” to with reasons for both.
+- A timeline for the development of Helm 4.
+A timeline sets expectations for the time of development.
+If the time for development is open-ended then development could end up in a perl 6 development cycle and just take forever.
+At some point development needs to stop.
+
+This HIP aims to define sucess criteria for these.
```

Comment:
Should we take out this section? I ask for a few reasons...

* "maintenance mode" can send a bit of the wrong impression because Helm has been active, even if the activity has been lower. This includes new features. I asked AI what "maintenance mode" means and this is what I got...
> "Maintenance mode" in reference to an open source project typically means that the project is no longer actively developed with new features or major changes. Instead, the focus is on maintaining the existing codebase.
* We don't have a full list of what's in scope so this pre-requisite isn't achieved and won't likely ever be.

Reply:
agree, removed (I copied from the doc "Prerequisites" section; but agree doesn't really belong in the HIP in hindsight)